libnmstate/appliers/bond.py

```diff
@@ -35,3 +80,21 @@ def is_in_mac_restricted_mode(bond_options):
     return BondMode.ACTIVE_BACKUP == bond_options.get(
         Bond.MODE
     ) and bond_options.get("fail_over_mac") in ("1", 1, "active",)
+
+
+def get_bond_named_option_value_by_id(option_name, option_id_value):
+    """
+    Given an option name and its value, return a named option value
+    if it exists.
+    Return the same option value as inputted if:
+    - The option name has no dual named and id values.
+    - The option value is not numeric.
+    - The option value has no corresponding named value (not in range).
+    """
+    option_value = BOND_OPTIONS_NUMERIC_TO_NAMED_MAP.get(option_name)
+    if option_value:
+        try:
+            return option_value[int(option_id_value)]
+        except (ValueError, IndexError):
```

Comment:
Can you use these one as you did in other file?

```python
with contextlib.suppress(ValueError, IndexError):
```

Reply:
Done